test_runner/performance/test_perf_olap.py

```diff
@@ -108,7 +125,28 @@ def run_psql(env: RemoteCompare, labelled_query: LabelledQuery, times: int) -> N
         log.info(f"Run {run}/{times}")
         with env.zenbenchmark.record_duration(f"{label}/{run}"):
             env.pg_bin.run_capture(["psql", connstr, "-c", query], env=environ)
+    if os.getenv('TEST_OLAP_COLLECT_EXPLAIN', 'false').lower() == 'true':
```

Comment:
I prefer to add an argument to the method (something like `explain: bool = False`), use it in the body, and have `os.getenv('TEST_OLAP_COLLECT_EXPLAIN', 'false').lower() == 'true'` part in a test itself.
This should eliminate the need of an additional `run_psql_once_without_explain` method

Reply:
Yes your suggestion is much better style (DRY)